docs/_components/tables.md

```diff
@@ -36,6 +36,33 @@ Table rows will accept the normal collection of state classes which will highlig
 * has-inverse
 * has-new
 
+## Tables with badges in "Actions" dropdown
+
+To add badges to the "Actions" dropdown when items are selected a class "has-badges" needs to be added in an element grouping both the dropdown and the table together.
```

Comment:
Would suggest noting that this could be a `div` or a `form` depending on what you need to do